opentelemetry-user-events-logs/src/logs/exporter.rs

```diff
@@ -161,10 +168,14 @@ impl UserEventsExporter {
         }
     }
 
-    /// Gets the event name from the log record
-    fn get_event_name<'a>(&self, _record: &'a opentelemetry_sdk::logs::SdkLogRecord) -> &'a str {
-        // TODO: Add callback to get event name from the log record
-        "Log"
+    /// Gets the event name from the log record using the provided callback or returns "Log" if no callback is set
+    fn get_event_name<'a>(&self, record: &'a opentelemetry_sdk::logs::SdkLogRecord) -> &'a str {
```

Comment:
This is confusing to me. The callback returns `&static str`, but, here we are returning the `&str` tied to the lifetime of `LogRecord`.